src/test.js

```diff
@@ -2,4 +2,8 @@ describe('peanutButter Tests', () => {
   it('singular test', () => {
     expect(2).toEqual(2);
   });
+
+  it('tests toContain', () => {
+    expect([1,2,3]).toContain(2);
```

Comment:
I don't think your Prettier is setup properly? `[1,2,3]` should've been formatted to `[1, 2 ,3]`.